internal/access/credential.go

```diff
@@ -73,3 +79,71 @@ func UpdateCredential(c *gin.Context, user *models.Identity, newPassword string)
 
 	return nil
 }
+
+func lowercaseCheck(n int, password string) bool {
+	match, _ := regexp.MatchString(fmt.Sprintf(`(.*\p{Ll}){%d,}`, n), password)
+	fmt.Println(match)
+	return match
+}
+
+func uppercaseCheck(n int, password string) bool {
+	match, _ := regexp.MatchString(fmt.Sprintf(`(.*\p{Lu}){%d,}`, n), password)
+	return match
+}
+
+func numberCheck(n int, password string) bool {
+	match, _ := regexp.MatchString(fmt.Sprintf(`(.*\p{N}){%d,}`, n), password)
+	return match
+}
+
+// list is from OWASP, wikipedia
+func symbolCheck(n int, password string) bool {
+	match, _ := regexp.MatchString(fmt.Sprintf(`(.*[ !"#$%%&'()*+,-./\:;<=>?@^_{}|~%s%s]){%d,}`, regexp.QuoteMeta(`/\[]`), "`", n), password)
```

Comment:
Might be better to negate letters and digits instead of listing out each symbol `[^a-zA-Z0-9]`